docs/Agent/user-guide/man_page.md

```diff
@@ -333,107 +265,205 @@ pbench-register-tool-trigger, Registers the tool trigger
 
 **DESCRIPTION**  
-Move result directories to the configured Pbench server.This command is used when pushing result tarball using token.
+
+Collects data from the registered tools while running a user-specified action. This can be a specific synthetic benchmark workload, a real workload, or simply a delay to measure system activity.
+
+Here are the steps involved
+
+- Invoking pbench-user-benchmark with your workload generator as an argument: that will start the collection tools on all the hosts
+- Next, it will run your workload generator; when that finishes, it will stop the collection tools on all the hosts
+- Finally, the postprocessing phase will gather the data from all the remote hosts and run the postprocessing tools on everything.
 
 **OPTIONS**  
- -C, --config PATH  
-Path to a pbench-agent configuration file (defaults to the '\_PBENCH_AGENT_CONFIG' environment variable,if defined) [required]
+`-C`, `--config` PATH  
+This option is required if not provided by the `_PBENCH_AGENT_CONFIG` environment variable.
+Path to the pbench-agent configuration file.
 
---controller STR  
-Override the default controller name
+`--tool-group`  
+The tool group to use for the list of tools
 
---token STR  
-pbench server authentication token [required]
+`--iteration-list`  
+A file containing a list of iterations to run for the provided script;
+the file should contain one iteration per line . With a leading '#' (hash) character used for comments and blank lines are ignored.
+Each iteration line should use alpha-numeric characters before the first space to name the iteration, with the rest of the line provided as arguments to the script;  
+_NOTE: --iteration-list is not compatible with --use-tool-triggers_
 
---delete / --no-delete  
-Remove local data after successful copy [default: delete]
+`--sysinfo` STR,[STR]  
+comma-separated values of system information to be collected; available: default, none, all, ara, block, insights, kernel_config, libvirt, security_mitigations, sos, stockpile, topology
 
---xz-single-threaded  
-Use single-threaded compression with 'xz'
+`--pbench-pre` STR  
+Path to the script which will be executed before tools are started  
+_NOTE: --pbench-pre is not compatible with --use-tool-triggers_
 
---show-server STR  
-Display information about the pbench server where the result(s) will be moved (Not implemented)
+`--pbench-post` STR  
+Path to the script which will be executed after tools are stopped and postprocessing is complete  
+_NOTE: --pbench-post is not compatible with --use-tool-triggers_
 
---help  
+`--use-tool-triggers`  
+Use tool triggers instead of normal start/stop around script;  
+_NOTE: --use-tool-triggers is not compatible with --iteration-list,--pbench-pre, or --pbench-post_
+
+`--no-stderr-capture`  
+Do not capture the stderr of the script to the result.txt file
+
+`--<command to run>`
+ It can be a script, or an executable, or a shell command 
```

Comment:
This is not a command _option_.  `<command-to-run>` is a command _argument_.  It must be specified (i.e., it is not "optional"), and it should appear on the command line after a double-hyphen with no option-name, which indicates the end of the options list (but, I think this is only necessary when the command to run also has options).

This argument should be described in the Description section.